app/controllers/tributes_controller.rb

```diff
@@ -14,6 +14,13 @@ def create
     end
   
     def update
+        update_tribute = Tribute.update(tribute_params)
```

Comment:
Look at the syllabus at the update endpoint on the wildlifetracker. We have to provide a params to the update endpoint.
https://github.com/learn-academy-2023-foxtrot/syllabus/blob/main/rails/rails-api.md